src/main/webapp/SearchResults.js

```diff
@@ -2,31 +2,47 @@
  * @param {query} input
  */
 async function searching(input) {
```

Comment:
I cant pass all for the check because of the async fiction.
I get: `Parsing error: Unexpected token function` and I'm not sure how to fix it.

Reply:
Looks like it was an eslint error. #26 should fix this so don't worry about this particular error